modules/researchsoftware/exercise-writing-software.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Writing Software
+type: exercise
+order: 2
+---
+
+## Writing software with Python (10 minutes)
+
+<img src="media/python-online.png" width="70%">
+<small>https://www.online-python.com/</small>
+
+### If you have never written software before:
+
+- Visit https://www.online-python.com/
+  - it will show you as an example the code for a function that adds items together.
+- Play around with the code a bit, e.g.
+  - Try using words instead of numbers as inputs. What happens and why?
+  - Add a function that subtracts items rather than adding them. What happens now if you use words instead of numbers, and why?
+  - What else can you do?
+
+### If you have experience in writing software:
+
```

Comment:
I really like how this exercise came out. I'm curious about seeing it in action.